docroot/sites/its.uiowa.edu/modules/its_core/src/Plugin/Block/AlertTypeLegend.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Drupal\its_core\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/**
+ * Provides an 'Alert Type Legend' block.
+ *
+ * @Block(
+ *   id = "alert_type_legend_block",
+ *   admin_label = @Translation("Alert Type Legend"),
+ *   category = @Translation("Site custom")
+ * )
+ */
+class AlertTypeLegend extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return ['label_display' => FALSE];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $build['alert_type_legend'] = [
+      '#type' => 'html_tag',
+      '#tag' => 'div',
+      '#attributes' => [
+        'class' => [''],
+      ],
+    ];
+    $build['alert_type_legend']['heading'] = [
+      '#type' => 'html_tag',
+      '#tag' => 'div',
+      '#attributes' => [
+        'class' => [
+          'headline',
+          'headline--uppercase',
+          'h5',
+        ],
+      ],
+    ];
+
+    $build['alert_type_legend']['heading']['content'] = [
+      '#type' => 'html_tag',
+      '#tag' => 'span',
+      '#value' => 'Alert types',
```

Comment:
We are using "Legend" on https://its.uiowa.ddev.site/alerts/calendar and https://its.uiowa.edu/alerts uses "Legend" as well. Do we want to keep the same heading text here to be consistent?